doc/sphinx/Clusters_from_Scratch/active-passive.rst

```diff
@@ -208,14 +217,14 @@ automatically, and no errors are reported.
 
     Two-node clusters are a special case. By the above definition,
     a two-node cluster would only have quorum when both nodes are
-    running. This would make the creation of a two-node cluster pointless,
-    but corosync has the ability to treat two-node clusters as if only one node
-    is required for quorum.
+    running. This would make the creation of a two-node cluster pointless.
+    But corosync has the ability to require only one node for quorum in a two-
+    node cluster.
```

Comment:
s/But/However,/ reads better imo

maybe worth mentioning that fencing prevents both nodes from grabbing quorum

Reply:
> But/however

Agreed.

> fencing/quorum

I thought about that and was unsure if that would be TMI. The next push will have this draft note:
```
    .. NOTE::

        You might wonder, "What if the nodes in a two-node cluster can't
        communicate with each other? Wouldn't this **two_node: 1** setting
        create a split-brain scenario, in which each node has quorum separately
        and they both try to manage the same cluster resources?"

        As long as fencing is configured, there is no danger of this. If the
        nodes lose contact with each other, each node will try to fence the
        other node. Resource management is disabled until fencing succeeds;
        neither node is allowed to start, stop, promote, or demote resources.

        After fencing succeeds, the surviving node can safely recover any
        resources that were running on the fenced node.

        If the fenced node boots up and rejoins the cluster, it does not have
        quorum until it can communicate with the surviving node at least once.
        This prevents "fence loops," in which a node gets fenced, reboots,
        rejoins the cluster, and fences the other node. This protective
        behavior is controlled by the **wait_for_all: 1** option, which is
        enabled automatically when **two_node: 1** is configured.
```

Part of the fencing chapter improvement on my to-do list is to include a fence delay, so we can skirt the question of both nodes getting fenced here.

Reply:
Sounds good